mypy/checkmember.py

```diff
@@ -368,33 +368,23 @@ def lookup_member_var_or_accessor(info: TypeInfo, name: str,
         return None
 
 
-def check_method_type(functype: FunctionLike, itype: Instance, is_classmethod: bool,
+def check_method_type(functype: FunctionLike, original_type: Type, is_classmethod: bool,
```

Comment:
Add docstring that at least explains the purpose of this method and the `original_type` argument.

Also, can you explain why you this change helps?